lighthouse-core/audits/byte-efficiency/unminified-javascript.js

```diff
@@ -8,9 +8,19 @@
 const ByteEfficiencyAudit = require('./byte-efficiency-audit');
 const esprima = require('esprima');
 
-const IGNORE_THRESHOLD_IN_PERCENT = .1;
+const IGNORE_THRESHOLD_IN_PERCENT = 10;
```

Comment:
lol looks like past audits use a mix:  ![image](https://user-images.githubusercontent.com/39191/34188299-8729b1f8-e4ea-11e7-87a3-d5ec52889837.png)

would be nice to have types for this  and milliseconds v seconds.  damn.

Reply:
lol, yeah I ended up switching because the `wastedPercent` value is x/100, it's the `wastedRatio` thats x/1